src/framework/filterControl/FilterControl.spec.jsx

```diff
@@ -55,4 +56,34 @@ describe('FilterControl', () => {
       });
     });
   });
+
+  describe('add button', () => {
```

Comment:
I think it is clearer to have two wrapping `describe` blocks – for props and for the add button. The first one tests that the props are used correctly, the second one tests that the UI behaves as expected.